forch/forchestrator.py

```diff
@@ -625,6 +641,7 @@ def main():
         http_server.map_request('host_path', forchestrator.get_host_path)
         http_server.map_request('list_hosts', forchestrator.get_list_hosts)
         http_server.map_request('sys_config', forchestrator.get_sys_config)
+        http_server.map_request('dva_state', forchestrator.get_dva_state)
```

Comment:
THis should not be a separate top-level API... it's just another attribute for a given port and/or MAC address. We already have plenty of entry points to get that information, so it should be an incremental addition to those places, not something new.

For every switch port (already existing API), we should be able to see which VLAN & ACL are applied.

For each MAC address (already existing API), we should be able to see what switch/port/VLAN/ACL it's associated with.

Reply:
Moved VLAN/ACLs into switch_state and list_hosts API.